CHANGELOG.md

```diff
@@ -9,8 +9,15 @@ The latest version is considered WIP and it is a subject of change. All other
 versions are frozen. To freeze current version all files are copied to a new
 version directory, and  then changes are introduced.
 
-## [v4.2.0] WIP
+## [v4.3.0] WIP
```

Comment:
Is it correct to change this information here? 
Or is this usually done in a separate PR?